src/components/PostsList.tsx

```diff
@@ -1,86 +1,72 @@
 import React from 'react';
+import { Post } from '../types/Post';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+type Props = {
+  userPosts: Post[];
+  handleShowComment: (post: Post) => void;
+  showSideBar: number | null;
+  setShowSideBar: (id: number | null) => void;
+  setShowCommentForm: (item: boolean) => void;
+};
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  userPosts,
+  handleShowComment,
+  showSideBar,
+  setShowSideBar,
+  setShowCommentForm,
+}) => {
```

Comment:
You are not describing objects in `propTypes`. It's a good practice to describe them to avoid bugs in the future.